hooks/pre-push.d/rs-spdx-license-test

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+top=`git rev-parse --show-toplevel`
+status=0
+for file in $(find "$top" -path "$top"/target -prune -o -name "*.rs")
+do
+    if [ -f "$file" ] && [ ! "$(sed -n '/^\/\/ SPDX-License-Identifier: Apache-2.0/p;q' "$file")" ]
```

Comment:
```suggestion
    if ! awk '/\/\/ SPDX-License-Identifier: Apache-2.0$/ { exit 0 } { exit 1 }' $file; then
```

https://en.wikipedia.org/wiki/AWK should help explain why this works. Basically, awk will try to match the first line against the pattern. If it succeeds, `exit 0` otherwise `exit 1` will be issued because no pattern matches everything.